src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,223 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class StringCalculator {
+
+    private static final String SPLIT_REGEX = " ";
+    private static final String MARK_ADD = "+";
+    private static final String MARK_SUBTRACTION = "-";
+    private static final String MARK_MULTIPLICATION = "*";
+    private static final String MARK_DIVISION = "/";
+    
+    private List<String> inputList;
+    private int baseValue = 0;
+    private int differentValue = 0;
+    private String mark;
+
+    public StringCalculator(String input) {
+        this.inputList = this.splitInput(input);
+    }
+
+    public List<String> getInputList() {
+        return inputList;
+    }
+
+    public void setInputList(List<String> inputList) {
+        this.inputList = inputList;
+    }
+
+    public int getBaseValue() {
+        return baseValue;
+    }
+
+    public void setBaseValue(int baseValue) {
+        this.baseValue = baseValue;
+    }
+
+    public int getDifferentValue() {
+        return differentValue;
+    }
+
+    public void setDifferentValue(int differentValue) {
+        this.differentValue = differentValue;
+    }
+
+    public String getMark() {
+        return mark;
+    }
+
+    public void setMark(String mark) {
+        this.mark = mark;
+    }
+
+    /**
+     * 입력 값을 쪼개어 리스트에 담습니다.
+     * @param input
+     * @return
+     */
+    public List<String> splitInput(String input) {
+        return Arrays.asList(input.split(StringCalculator.SPLIT_REGEX));
+    }
+
+    /**
+     * 입력 된 문자열 값을 계산합니다.
+     * @return
+     */
+    public int calculate() throws IllegalArgumentException {
+        for (int i = 0 ; i < this.getInputList().size() ; i++) {
```

Comment:
```suggestion
        for (int i = 0 ; i < inputList.size() ; i++) {
```
getter 메서드 호출해서 사이즈 받아올 필요가 있을까요?
아래 코드도 마찬가지입니다
inputList는 인스턴스 변수이기 때문에 getter로 호출할 필요가 없습니다
불필요하게 getter를 쓰는 관련 코드는 수정해보아요